api/app/src/routes/medical/link.ts

```diff
@@ -68,22 +68,22 @@ router.delete(
  * Builds and returns the current state of a patient's links across HIEs.
  *
  * @param   req.params.patientId     Patient ID for which to retrieve links.
- * @return  {PatientLinks}          The patient's current and potential links.
+ * @param   req.query.facilityId     The ID of the facility to provide the NPI to get links for patient.
+ * @return  {PatientLinks}           The patient's current and potential links.
```

Comment:
nit: as per the [tsdoc convention](https://tsdoc.org/pages/tags/returns/), we can just have do the following:

` * @returns The patient's current and potential links.`

Reply:
I was thinking about this yesterday... this makes sense for functions, but I wonder if we should have the type for endpoints. Reasoning being, doc generation likely won't be able to tell (easily) what the result type is.

Thoughts?

Reply:
Note that depending on how we do doc gen, these comments could be auto-generated as part of the API interface. AFAIK, there is no good way for us to do doc/SDK gen for what we need strictly off of the tsdoc.

As such, for now we could just be consistent with our own comment style (adhering to tsdoc)?